specification/cluster/put_component_template/ClusterPutComponentTemplateRequest.ts

```diff
@@ -20,7 +20,7 @@
 import { RequestBase } from '@_types/Base'
 import { Metadata, Name, VersionNumber } from '@_types/common'
 import { Duration } from '@_types/Time'
-import { IndexState } from '@indices/_types/IndexState'
+import { IndexTemplateMapping } from '@indices/put_index_template/IndicesPutIndexTemplateRequest'
```

Comment:
This import isn't particularly pretty. I'll leave it up to you whether this is fine or if we should extract the `IndexTemplateMapping` class to a dedicated file. If we're extracting it (which I'm in favor of), we should probably also rename it.

Reply:
Yep, moving it to `specification/cluster/_types` works for me :+1:. I'm having trouble coming up with a good name though... `IndexTemplate` is already used for retrieving templates. Is there some kind of naming convention for types that are used for PUT requests?